cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -121,7 +138,7 @@ public KafkaCruiseControlServlet(KafkaCruiseControl kafkaCruiseControl) {
    */
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-    ACCESS_LOG.info("Received {}, {} from {}", urlEncode(request.toString()), urlEncode(request.getRequestURL().toString()),
+    ACCESS_LOG.info("Received {}, {} from {}", request.toString(), request.getRequestURL().toString(),
```

Comment:
We have to always use the url encoding for request logging for security purpose.

Reply:
its hard to understand from logs what the actual url is, if we urlencode and store it in the logs. i am reverting it to urlencoded version